packages/lesswrong/components/form-components/TagMultiselect.tsx

```diff
@@ -22,12 +22,13 @@ const styles = (theme: ThemeType): JssStyles => ({
   },
 });
 
-const TagMultiselect = ({ value, path, classes, label, placeholder, updateCurrentValues }: {
+const TagMultiselect = ({ value, path, classes, label, placeholder, hidePostCount=false, updateCurrentValues }: {
```

Comment:
For the tag filter on the search page, showing the post count is misleading since that doesn't correspond with the number of matching search results.